qt/place_page_dialog.cpp

```diff
@@ -84,6 +85,34 @@ PlacePageDialog::PlacePageDialog(QWidget * parent, place_page::Info const & info
     addEntry(DebugPrint(PropID::FMD_CUISINE), cuisines);
 
   QDialogButtonBox * dbb = new QDialogButtonBox();
+
+  QPushButton * toButton = new QPushButton("To");
+  toButton -> setIcon(QIcon(":/navig64/point-finish.png"));
+  connect(toButton, &QAbstractButton::clicked, this,
+          [this]() {
+            this->SetRoutePointAddMode(RouteMarkType::Finish);
+            this->OnClose();
+            });
```

Comment:
There is no need to introduce yet another different formatting here and below. Note `this` syntax. 
```suggestion
  connect(toButton, &QAbstractButton::clicked, this, [this]
  {
    SetRoutePointAddMode(RouteMarkType::Finish);
    OnClose();
  });
```